### PR TITLE
Matrix fix

### PR DESCRIPTION
**Pull request by ceco-fmedia**
Fixed Matrix.setTransform gives different result from Transform.updateLocalTransform.

`Return rotation instead of skew if skewX+skewY is slightly below 360 degrees.`

### CODE REVIEW

**Comment by ivanpopelyshev** [comment]
I dont know why do we need this method anyway :)

**Comment by cursedcoder** [comment]
Adding a test case for that fix is highly appreciated.

**Comment by ivanpopelyshev** [comment]
I can't double-approve :(

File: src/core/math/Matrix.js [hunk 1]
```diff
@@ -363,7 +351,7 @@ export default class Matrix
 
         const delta = Math.abs(skewX + skewY);
 
-        if (delta < 0.00001)
+        if (delta < 0.00001 || Math.abs((2 * Math.PI) - delta) < 0.00001)
```
Comment (bigtimebuddy) [inline src/core/math/Matrix.js]: Minor thing, could you use import `PI_2` from **src/core/const.js** instead of using `2 * Math.PI`?

**Comment by ivanpopelyshev** [comment]
We need to fix the random seed. Is there any way to do it in JS? If scale is near zero, transform will go fubar.

**Comment by ivanpopelyshev** [comment]
I think we have to add [seedrandom](https://github.com/davidbau/seedrandom) to dev-dependencies and use it there.

**Comment by GoodBoyDigital** [comment]
you da man @ceco-fmedia 👍

**Pull request merged**